VERSION

```diff
@@ -0,0 +1 @@
+2.5.0
```

Comment:
question:
maybe we should start with `0.1.0` or `1.0.0`. what do you think?
this is the version of this piece of code after all.